db/migrate/20210219212931_change_cursor_to_string.rb

```diff
@@ -12,7 +12,7 @@ def up
 
   def down
     change_table(:maintenance_tasks_runs) do |t|
-      t.change(:cursor, :bigint)
+      t.change(:cursor, "bigint USING cursor::bigint")
```

Comment:
This is postgresql specific, right? Which makes it broken for any other database. We should look for a solution that works for all adapters or branch in the worst case

Reply:
Ohhh, you're right, I forgot about that. I think the CAST function should be cross-platform.